tests/e2e/specs/add-paid-campaigns/add-paid-campaigns.test.js

```diff
@@ -499,6 +511,20 @@ test.describe( 'Set up Ads account', () => {
 	} );
 
 	test.describe( 'Create Ads with billing data already setup', () => {
+		test.beforeAll( async () => {
+			// Reset the campaigns set in the previous tests.
+			await setupBudgetPage.fulfillAdsCampaignsRequest( [] );
+
+			// Mock budget recommendations.
+			await setupBudgetPage.fulfillRequest(
```

Comment:
@kt-12 Why we need to set the budget recommendation ? 🤔

Reply:
Without budget recommendations, the BudgetRecommendation component was throwing an error when the internal getHighestBudget() function runs recommendations.reduce() because recommendations is undefined. I couldn't reproduce this condition so am not sure it's a realistic scenario, since so much of the state is getting mocked previous to this running. Mocking this as well resolves the issue.